cmd/protoc-gen-fastmarshal/templates/singlefile.go.tmpl

```diff
@@ -57,10 +57,10 @@ func (m *{{ .GoIdent.GoName }}) Marshal() ([]byte, error) {
 func (m *{{ .GoIdent.GoName }}) MarshalTo(dest []byte) error {
 	var (
     	enc = csproto.NewEncoder(dest)
-		buf []byte
-		err error
+    	buf []byte
+    	err error
 	)
-	_, _ = buf, err // ensure no unused variables
+	_, _, _ = enc, buf, err // ensure no unused variables
```

Comment:
Need to make this same change in `.../template/permessage.go.tmpl`.  I'd like to consolidate `singlefile.go.tmpl` and `permessage.go.tmpl` at some point, but it's not there today.